dist/styles/base.scss

```diff
@@ -248,6 +248,11 @@
   }
 }
 
+// Include a "hidden option" class that can be placed on empty placeholder options to keep them out of the dropdown
+.hidden_option {
```

Comment:
Agreed on both of those. I'll update the name with prefix -- I would guess `fr_` is the prefix for formrenderer, so good call.